modules/service/src/main/scala/ch/epfl/bluebrain/nexus/admin/service/Main.scala

```diff
@@ -85,14 +86,22 @@ object Main {
       val projects                                   = Projects(agg, blazegraphClient)
       val api = uriPrefix(appConfig.http.apiUri)(
         ProjectRoutes(projects).routes ~ ProjectAclRoutes(projects, AkkaStream()).routes)
+      val staticResourceRoutes = new StaticResourceRoutes(
+        Map("/contexts/project" -> "/project-context.json",
+            "/contexts/filter"  -> "/filter-context.json",
+            "/schemas/project"  -> "/schemas/nexus/core/project/v0.1.0.json"),
+        "static",
+        appConfig.http.apiUri
```

Comment:
It seems weird to me that the value of the `@id` is related to the `apiUri`. That would mean different deployments have different ids for the same resource.
I think we are again mixing the `@id` of the thing with its accessibility

Reply:
As far as I understand it , `@id` should be resolvable. I'm not sure how to do it otherwise without relating it to `apiUri`, unless we want to host it on a static page (e.g. github pages?) accessible to all the deployments?

Reply:
Our only option at this point is to use an id derived from the underlying host. Let's merge this as is and address the issue across all projects.